splitio/angular-sdk-examples
# Update Angular version & switch to SDK-browser

# angular-sdk-examples

## What did you accomplish?
In this PR I'm introducing 2 big changes:

1 - Upgrade Angular version in the example, from 9 to 11.2.5
2 - Removed the old SDK client in favour of  the new SDK-browser version.

... and other small changes:
- Added `.nvmrc` file
- Upgraded other packages/dependencies
- Some fixes/additions in the documentation

Also, in the example we are using `InLocalStorage` capability. You should be able to see it working when correctly setting the example with a real APIKEY and key.

<img width="812" alt="Screen Shot 2021-03-17 at 17 22 55" src="https://user-images.githubusercontent.com/80265540/111533924-213c6a00-8746-11eb-837c-688331c1e858.png">

## How do we test the changes introduced in this PR?
Following the same instructions as the README states, to build and run the local app.

_Expected behaviour:_ You should see the app running and the Features being requested.

## Related tickets:
[Update Angular Example](https://splitio.atlassian.net/browse/SDKS-2479)
[Update Angular Example with new SDK Browser](https://splitio.atlassian.net/browse/SDKS-2480)

## Extra Notes
Fixing [this issue brought by a customer](https://github.com/splitio/angular-sdk-examples/pull/9)
